psamm/commands/sbmlexport.py

```diff
@@ -24,6 +24,15 @@
 class SBMLExport(MetabolicMixin, Command):
     """Export model as SBML file."""
 
+    @classmethod
+    def init_parser(cls, parser):
+        parser.add_argument(
+            'file', type=str, help='File path for writing the Excel workbook',
+            default='-')
```

Comment:
Should call the super method as the last thing in this method. See other commands for example.